src/js/App/Header/LogoutAlert.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
+import cookie from 'js-cookie';
+
+class LogoutAlert extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            alertOneVisible: cookie.get('justLoggedOut') === 'true',
+            delay: 5000
+        };
+    }
+
+    hideAlertOne = () => {
+        this.setState({ alertOneVisible: false });
+        cookie.set('justLoggedOut', 'false');
+    }
+    componentDidMount() {
+        this.setTimer();
+    }
+
+    setTimer = () => {
+        if (this._timer !== null) {
+            clearTimeout(this._timer);
+        }
+
+        // hide after `delay` milliseconds
+        this._timer = setTimeout(function() {
+            this.setState({ alertOneVisible: false });
+            this._timer = null;
+        }.bind(this), this.state.delay);
+    }
+
+    render() {
+        const { alertOneVisible } = this.state;
+        return (
+            <React.Fragment>
+                { alertOneVisible && (
+                    <Alert
+                        variant="success"
+                        title="Congratualtions you have successfully logged out!"
```

Comment:
Is this the string that the mocks have?

Otherwise, i'd just do `You have logged out.`

Reply:
https://projects.engineering.redhat.com/browse/RHCLOUD-1810 the 'Congratulations' isn't but the rest is